src/components/PostCard/PostCompletion/PostCompletion.js

```diff
@@ -5,37 +5,67 @@ import DropdownButton from 'components/DropdownButton'
 export default function PostCompletion ({ type, startTime, endTime, isFulfilled, fulfillPost, unfulfillPost }) {
   var label
 
+  const promptOptions = {
+    request: 'Do you still need this?',
```

Comment:
so I've noticed while using Hylo that this one doesnt always feel right because sometimes a request is on behalf of someone else so its not "you" still needing it. While we are working on this code, what do you think about changing to "Is this still needed?"

Reply:
love it